server/src/main/java/org/opensearch/indices/recovery/RecoverySettings.java

```diff
@@ -65,6 +65,13 @@ public class RecoverySettings {
         Property.NodeScope
     );
 
+    public static final Setting<ByteSizeValue> SEGREP_MAX_BYTES_PER_SEC_SETTING = Setting.byteSizeSetting(
```

Comment:
Hey @Ferrari248, thanks for taking the initiative with this! 

Could you add a couple scenario tests to make sure everything is as expected? I am not an expert on segRep but I would be interested in seeing whether you could confirm the change in behavior through modifying this setting. 

Another thing to consider is that in order to backport this to 2.x, I think we will want to make sure that users who do not configure this new setting maintain the old behavior. I see below you decide between the rateLimiter and your knew limiter. If we could make sure that we use the rateLimiter/the same setting as the rateLimiter, whenever the new limiter is not present that would be great.

Reply:
Thanks for your reply.
> Another thing to consider is that in order to backport this to 2.x, I think we will want to make sure that users who do not configure this new setting maintain the old behavior.

Good idea, ok. Let me try.
> Could you add a couple scenario tests

Sorry, I don't know where and how to add? Do you mean Unit Test?

Reply:
I added one setting "indices.replication.use_individual_rate_limiter" defaults to `false` , if set to `true` , the "indices.replication.max_bytes_per_sec" setting will take effect.